players/group5/player_map.py

```diff
@@ -214,4 +225,231 @@ def update_map(self, turn_num: int, percept: TimingMazeState):
 
     def get_seen_counts(self, relative_coords: List[List[int]]) -> List[int]:
         return [self._cell_seen_count.get(tuple(self._get_map_coordinates(c)), 0) if isinstance(c, list) else 0 for c in relative_coords]
+    
+    def get_valid_moves(self) -> List[int]:
+        # Not implemented
+        pass
+
+    def get_wall_freq_candidates(self, door_id: DoorIdentifier) -> List[Set[int]]:
+        # Not implemented
+        pass
+
+"""
+StartPosCentricPlayerMap is similar to SimplePlayerCentricMap but is not centric on the user's current position but rather the first start position which we assign to a constant coordinate (i.e., it receives/outputs start position-centric coordinate data). 
```

Comment:
this class is what we are using now. (ABSOLUTE/Start Position Centric Map) @Yoni-Livstone @Jinha-Park @eylam